code/modules/atmospherics/gasmixtures/reaction_factors.dm

```diff
@@ -175,33 +178,35 @@
 		/datum/gas/hydrogen = "Hydrogen is consumed at 2 reaction rate",
 		/datum/gas/proto_nitrate = "Proto-Nitrate is produced at 2.2 reaction rate",
 		"Energy" = "[PN_FORMATION_ENERGY] joules of energy is released per reaction rate",
-		"Temperature" = "Can only occur between [PN_FORMATION_MIN_TEMPERATURE] - [PN_FORMATION_MAX_TEMPERATURE] kelvin. Higher temperature increases the reaction rate.",
+		"Temperature" = "Can only occur between [PN_FORMATION_MIN_TEMPERATURE] - [PN_FORMATION_MAX_TEMPERATURE] kelvin. Reaction rate peaks at [PN_FORMATION_LOWER_TEMPERATURE_MAXIMUM_EFFICIENCY] and [PN_FORMATION_UPPER_TEMPERATURE_MAXIMUM_EFFICIENCY] Kelvin, and drops at [PN_FORMATION_MIN_TEMPERATURE], 7,500, and [PN_FORMATION_MAX_TEMPERATURE] Kelvin.",
```

Comment:
This is really runon, what's it trying to get across?

Reply:
I'll simplify the reaction itself in the next commit, along with description.

Reply:
The reaction itself and thhe description has been simplified.